massgen/frontend/displays/textual_terminal_display.py

```
@@ -4583,15 +4612,49 @@ def _refresh_input_modes_row_layout(self) -> None:
 
             width = self.size.width or (self.app.size.width if self.app else 120)
             plan_mode = getattr(getattr(self, "_mode_state", None), "plan_mode", "normal")
+            if self._mode_bar:
+                try:
+                    plan_mode = self._mode_bar.get_plan_mode()
+                except Exception:
+                    pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)